soc/arm/nordic_nrf/nrf52/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 zephyr_sources(
-  power.c
   soc.c
   )
 
-zephyr_sources_ifdef(CONFIG_ARM_MPU mpu_regions.c)
+zephyr_sources_ifdef(CONFIG_SYS_POWER_MANAGEMENT
+  power.c
+  )
+
+zephyr_sources_ifdef(CONFIG_ARM_MPU
+  mpu_regions.c
+)
```

Comment:
Add two spaces before `)` to stay consistent with the indentation used above.

Reply:
Fixed